NWChem.py

```diff
@@ -167,11 +167,11 @@ def GetPrerunNMRCalcs(Isomers):
 
     jobdir = os.getcwd()
     os.chdir('nmr')
-    """
+    
     for iso in Isomers:
-        iso.NMRInputFiles = glob.glob(iso.BaseName + 'ginp*com')
-        iso.NMROutputFiles.extend([x[:-4] + '.out' for x in iso.NMRInputFiles if IsGausCompleted(x[:-4] + '.out')])
-    """
+        iso.NMRInputFiles = sorted(glob.glob(iso.BaseName + '*.nw'))
+        iso.NMROutputFiles = sorted(glob.glob(iso.BaseName + '*.nwo'))
```

Comment:
I think we should still only include outputfiles that are valid (ie, have True return value from IsNWChemCompleted.)

Reply:
I have reverted this line.